app/models/miq_provision_virt_workflow.rb

```diff
@@ -339,8 +339,8 @@ def allowed_templates(options = {})
       end
     end
 
-    # Only select the colums we need
-    vms = vms.select(:id, :type, :name, :guid, :uid_ems, :ems_id, :cloud_tenant_id)
+    # Only select the colums we need on non-deprecated templates
```

Comment:
```suggestion
    # Only select the columns we need on non-deprecated templates
```

😑

Reply:
come on, that's a good typo I wanted to keep around 🤣